assets/state-mig-manager/0400_configmap.yaml

```
@@ -53,6 +53,78 @@ data:
           mig-devices:
             "7g.40gb": 1
 
+      # RTX-PRO-6000-96GB
+      all-1g.24gb+gfx:
+        - device-filter: ["0x2bb510DE"]
+          devices: all
```

To align with our other `all-*` configs, let's not specify a device filter here. Note, the same MIG profile can be supported on more than one device.
```suggestion
        - devices: all
```